experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -554,10 +554,12 @@ describe('HttpInstrumentation', () => {
       }
 
       it('should have 1 ended span when request throw on bad "options" object', () => {
-        assert.throws(() => http.request({ headers: { cookie: undefined} }), err => {
+        assert.throws(() => http.request({ headers: { cookie: undefined} }), (err: unknown) => {
           const spans = memoryExporter.getFinishedSpans();
           assert.strictEqual(spans.length, 1);
 
+          assert.ok(err instanceof Error);
```

Comment:
Types changed callback parameter from `Error` to `unknown`. Added an explicit type parameter and this check to assert it is really an error.